### ISSUE TITLE
Script deletes itself and text files, but not audio file

### ISSUE THREAD

**Issue by IAGOnDiscord** [Mcharlsto/rat#5]
Self explanatory.

There are two options on how to deal with this.
1. Don't delete any files
    - Can completely remove the ~~escape key~~ ***interrupt*** trap and its corresponding function
    - As such, can change shebang back to /bin/sh
    - May require a significant rework, but should be cleaner
2. Delete `audio.opus` as well
    - Easier to implement
    - Still requires Bash

While I personally think option 1 is better in the long run, it would require a significant rework that I'm not entirely sure how to approach.
Option 2 is suitable as a temporary fix. It would just require adding `rm audio.opus` to the `ctrl_c` function in `script.sh`.

**Comment by Mcharlsto** [comment]
When executing `rat` it deletes the entire folder, so it's not a problem there. The easiest way to do things as far as I can see would be just to remove the rm commands (and in turn the ctrl_c trap) from the script all together, as it's not necessary outside of the `rat` archive anyway. I'll try that now.

**Comment by IAGOnDiscord** [comment]
Thanks!

**Comment by IAGOnDiscord** [comment]
Also, for future reference, it's possible to use an exit trap instead of an interrupt trap.
```bash
trap <function> exit
```

**Comment by Mcharlsto** [comment]
I've reworked it all a bit - check 6fe51ea7c9390c94acbc05e3b7b3df91fd382b78. I removed the interrupt trap from `script.sh` and moved it all to the `rat` executable instead, so `script.sh` now has the original shebang and runs in ash etc. (still haven't fixed the weird glitch where it shows the audio progress bar but I'll get around to it).

`rat` automatically cleans up the whole temp folder anyway, so this issue seems resolved.
